client/homebrew/editor/editor.jsx

```diff
@@ -111,6 +111,10 @@ const Editor = createClass({
 			const customHighlights = codeMirror.getAllMarks().filter((mark)=>!mark.__isFold); //Don't undo code folding
 			for (let i=0;i<customHighlights.length;i++) customHighlights[i].clear();
 
+			let x = 0;
```

Comment:
improve this variable name please

Reply:
Ah yes, it's leftover from when the variable was much closer to the rest of the function and thus a little clearer in intent. I will fix. Was just glad I got all my console.logs out